### ISSUE TITLE
Checkpoints link to AMU SARS-CoV-2 in vitro is broken

### ISSUE THREAD

**Issue by miquelduranfrigola** [chemprop/chemprop#166]
Hi!

Checkpoints link to AMU SARS-CoV-2 in vitro model appears to be broken: http://chemprop.csail.mit.edu/checkpoints/download/4

I am getting a 500 Internal Server Error.

Thanks!
M

**Comment by hesther** [comment]
Yep, the link is broken, I am contacting the respective people to fix it.

**Comment by cjmcgill** [comment]
I believe this was resolved.

**Comment by miquelduranfrigola** [comment]
Hi! Apologies for reopening this issue. The link seems to be broken still? http://chemprop.csail.mit.edu/checkpoints/download/4
